src/nxt_http_route.c

```diff
@@ -1947,6 +1947,14 @@ nxt_http_route_addr_pattern_match(nxt_http_route_addr_pattern_t *p,
         break;
 #endif
 
+#if (NXT_HAVE_UNIX_DOMAIN)
+    case AF_UNIX:
+
+        match = (base->addr_family == AF_UNIX
+                 || base ->addr_family == AF_UNSPEC);
```

Comment:
Hm... doesn't it look strange if `*:80` will match any of Unix domain sockets as well?  This is also a significant behaviour change as currently it doesn't match Unix sockets.  `AF_UNSPEC` was used to match both IPv4 and IPv6, but not Unix sockets.

Note also a surplus space after the `base` in `base ->addr_family`.

Reply:
Ahhh, I was very confused here.  Will fix it in a moment.